sentry-delayed_job/lib/sentry/delayed_job/plugin.rb

```diff
@@ -40,8 +49,15 @@ def self.generate_contexts(job)
           queue: job.queue,
           created_at: job.created_at,
           last_error: job.last_error&.byteslice(0..1000),
-          handler: job.handler&.byteslice(0..1000)
+          handler: job.handler&.byteslice(0..1000),
+          handler_class: job.payload_object.class,
         }
+        if job.payload_object.is_a? Delayed::PerformableMethod
```

Comment:
I prefer having these computation in a `compute_job_class(payload_object)` method and at here we just do:
```ruby
  handler_class: job.payload_object.class,
  job_class: compute_job_class(job.payload_object)
}
```

also, can you write test cases for these changes too? thx